sample-apps/everything-jakarta-app/build.gradle

```diff
@@ -8,21 +8,24 @@ dependencies {
   //TODO update to 6.1 once using Jakarta EE 10
   providedCompile 'org.eclipse.microprofile:microprofile:5.0'
 
-  compile 'org.mongodb:mongo-java-driver:3.11.2'
+  implementation 'org.mongodb:mongo-java-driver:3.11.2'
 
-  testCompile project(':microshed-testing-testcontainers')
-  testCompile project(':microshed-testing-core-jakarta')
+  testImplementation project(':microshed-testing-testcontainers')
+  testImplementation project(':microshed-testing-core-jakarta')
+//  TODO solve why needed also regular core to compile?
+  testImplementation project(':microshed-testing-core')
```

Comment:
This shouldn't be necessary. I can do some testing to figure out why this working working.

Reply:
Again this points out my issue as described on your comment in Liberty sample app. I think we should use some other dependecy scope in the core and the modules to resolve this issues.